clarifai/client/lister.py

```diff
@@ -12,22 +12,30 @@ class Lister(BaseClient):
   def __init__(self, page_size: int = 16):
     self.default_page_size = page_size
```

Comment:
Is this still needed ?

Reply:
Yes. For scenarios where per_page is not set and the page is set, the user can set it in init itself. And also search class makes use of this..